test/js/birds.test.js

```diff
@@ -0,0 +1,35 @@
+import { active, authorPage, getPages } from '#/birds';
+
+import { collection2 } from './utils';
+
+describe('getPages', () => {
+  test('returns pages that the "bird" has authored', () => {
+    expect(getPages(collection2, 'miriam', true)).toEqual([collection2[0]]);
+  });
+  test('returns "oddbird" authored pages', () => {
+    const expected = [collection2[1], collection2[2]];
+
+    expect(getPages(collection2, 'erica')).toEqual(expected);
+  });
+});
+
+describe('active', () => {
+  test('filters pages from inactive birds', () => {
```

Comment:
maybe clarify that these are not bird-authored pages, but bird-detail pages?

```suggestion
  test('filters inactive bird-detail pages', () => {
```